README.md

```diff
@@ -3,4 +3,15 @@ Vulkan Capture for OBS
 
 This is a plugin for OBS Studio that allows Vulkan/OpenGL applications to be captured. For instructions on how to use, see [this page](https://github.com/nowrep/obs-vkcapture#usage).
 
-For this plugin to work you also need the corresponding tools to start the application with. These tools are available [on Flathub as `org.freedesktop.Platform.VulkanLayer.OBSVkCapture`](https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.OBSVkCapture).
\ No newline at end of file
+For this plugin to work you also need the corresponding tools to start the application with. These tools are available [on Flathub as `org.freedesktop.Platform.VulkanLayer.OBSVkCapture`](https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.OBSVkCapture).
+
+Installation
+-----------
+
+This plugin is an extension for OBS Studio therefore is not listed on the Flathub website to download it.
```

Comment:
I don't think this is grammatically correct. I would change it to

> This flatpak is an extension for OBS Studio and is therefore not listed on the Flathub website.

I would also drop the download part, since that is kind of implied here.

Reply:
Text replaced in the last commit with the one you provided